ui/components/forms/flavors/ncino/index-bs3.scss

```diff
@@ -172,4 +172,13 @@
   input[type="date"]::-webkit-input-placeholder{
     visibility: hidden !important;
   }
+  a.form-control.lookup {
+    width: auto;
+    height: auto;
+    position: absolute;
+    top: 5px;
+    right: 5px;
+    padding: 0;
```

Comment:
Would you please use tokens instead of fixed numeric values?  $spacing-none is 0.  $spacing-xx-small is 4px, but should be close enough for the top and right positions.  All SLDS tokens are documented at https://www.lightningdesignsystem.com/resources/tokens/.